contextbasedrestrictionsv1/context_based_restrictions_v1_examples_test.go

```diff
@@ -0,0 +1,461 @@
+	}
+}
+
+var _ = Describe(`contextbasedrestrictionsv1.ContextBasedRestrictionsV1 Examples Tests`, func() {
+	Describe(`External configuration`, func() {
+		It("Successfully load the configuration", func() {
+			var err error
+			_, err = os.Stat(externalConfigFile)
+			if err != nil {
+				Skip("External configuration file not found, skipping tests: " + err.Error())
+			}
+
+			os.Setenv("IBM_CREDENTIALS_FILE", externalConfigFile)
+			config, err = core.GetServiceProperties(contextbasedrestrictionsv1.DefaultServiceName)
+			if err != nil {
+				Skip("Error loading service properties, skipping tests: " + err.Error())
+			}
+
+			testAccountID = config["TEST_ACCOUNT_ID"]
+			if testAccountID == "" {
+				Skip("Unable to load TEST_ACCOUNT_ID configuration property, skipping tests")
+			}
+
+			testServiceName = config["TEST_SERVICE_NAME"]
+			if testServiceName == "" {
+				Skip("Unable to load TEST_SERVICE_NAME configuration property, skipping tests")
+			}
+
+			configLoaded = len(config) > 0
+		})
+	})
+
+	Describe(`Client initialization`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It("Successfully construct the service client instance", func() {
+			var err error
+
+			// begin-common
+
+			contextBasedRestrictionsServiceOptions := &contextbasedrestrictionsv1.Options{}
+
+			contextBasedRestrictionsService, err = contextbasedrestrictionsv1.NewContextBasedRestrictionsV1UsingExternalConfig(contextBasedRestrictionsServiceOptions)
+
+			if err != nil {
+				panic(err)
+			}
+
+			// end-common
+
+			Expect(contextBasedRestrictionsService).ToNot(BeNil())
+		})
+	})
+
+	Describe(`contextbasedrestrictionsv1.ContextBasedRestrictionsV1 request examples`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`CreateZone request example`, func() {
+			fmt.Println("\nCreateZone() result:")
+			// begin-create_zone
+
+			addressModel := &contextbasedrestrictionsv1.AddressIPAddress{
+				Type:  core.StringPtr("ipAddress"),
+				Value: core.StringPtr("169.23.56.234"),
+			}
+
+			createZoneOptions := contextBasedRestrictionsService.NewCreateZoneOptions()
+			createZoneOptions.SetName("SDK TEST - an example of zone")
+			createZoneOptions.SetAccountID(testAccountID)
+			createZoneOptions.SetDescription("SDK TEST - this is an example of zone")
+			createZoneOptions.SetAddresses([]contextbasedrestrictionsv1.AddressIntf{addressModel})
+
+			zone, response, err := contextBasedRestrictionsService.CreateZone(createZoneOptions)
+			if err != nil {
+				panic(err)
+			}
+			b, _ := json.MarshalIndent(zone, "", "  ")
+			fmt.Println(string(b))
+
+			// end-create_zone
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(201))
+			Expect(zone).ToNot(BeNil())
+			zoneID = *zone.ID
+
+		})
+		It(`ListZones request example`, func() {
+			fmt.Println("\nListZones() result:")
+			// begin-list_zones
+
+			listZonesOptions := contextBasedRestrictionsService.NewListZonesOptions(
+				testAccountID,
```

Comment:
I'd like to ask you to rename this variable to `accountId` if it doesn't conflict. The point is not displaying `test` in the reference.

Reply:
OK. No conflicts.